pages/add/form/detail-info-fields.js

```diff
@@ -1,26 +1,11 @@
-import React, { Component } from 'react';
```

Comment:
`pages/add.jsx` and `pages/profile-edit.jsx` both have this field in the form. I extracted the code so this lives on its own file.